data_prep/common/src/bin/generate_od.rs

```diff
@@ -2,36 +2,61 @@ use std::collections::{BTreeMap, BTreeSet};
 use std::process::Command;
 
 use anyhow::{bail, Result};
+use argh::FromArgs;
 use geo::{Intersects, MultiPolygon};
 use serde::Deserialize;
 use utils::Mercator;
 
 use backend::od::{DemandModel, ZoneID};
-use data_prep::StudyArea;
+use common_data_prep::StudyArea;
+
+#[derive(FromArgs)]
+/// Generate travel demand (OD) model
+struct Args {
+    /// path to study area boundaries.geojson, with a `kind` and `name` property
+    #[argh(option)]
+    study_area_boundaries: String,
+
+    /// path to a zones.geojson, with a `name` property
+    #[argh(option)]
+    od_zones: String,
+
+    /// path to an od.csv, with fields `zone1`, `zone2`, `count`
+    #[argh(option)]
+    od_csv: String,
+
+    /// path to the directory where gzipped output will be written, as
+    /// `{study_area.kind}_{study_area.name}.bin.gz`
+    #[argh(option)]
+    out_dir: String,
+}
 
-/// Generate travel demand (OD) model, and write to file
 fn main() -> Result<()> {
-    let study_areas = StudyArea::read_all_from_file()?;
+    let args: Args = argh::from_env();
+
+    let study_areas = StudyArea::read_all_from_file(&args.study_area_boundaries)?;
 
     let input_zones: Vec<Zone> = geojson::de::deserialize_feature_collection_str_to_vec(
-        &std::fs::read_to_string("zones.geojson")?,
+        &fs_err::read_to_string(&args.od_zones)?,
```

Comment:
Also switched to this, so we get better error messages with the path included for free